lib/cmds/files_cmds/upload.js

```diff
@@ -197,4 +210,44 @@ exports.handler = async argv => {
       fs.unlinkSync(file);
     }
   }
+
+  function getPathToStrip (stripPath, filePath, isDir) {
+    if (!stripPath) {
+      return '';
+    }
+
+    const baseFileName = normalizeFileName(filePath);
+    let lastSepPath = baseFileName.lastIndexOf('/');
+
+    if (isDir) {
+      lastSepPath = baseFileName.length;
+    }
+
+    const pathToStrip = filePath.substring(0, lastSepPath + 1);
+
+    console.log('filePath=' + filePath + '\npathToStrip = ' + pathToStrip);
+    return pathToStrip;
+  }
+
+  function getFileOnlyName (pathToStrip, fileName) {
+    if (pathToStrip.length > 0) {
+      return fileName.substring(pathToStrip.length);
+    } else {
+      return fileName;
+    }
+  }
+
+  function normalizeFileName (file) {
+    return (file.startsWith('./') ? file.slice(2) : file).replace(/\\/g, '/');
+  }
+  function getRemoteFileName (remotePath, fileOnlyName) {
+    if (!remotePath) {
+      return fileOnlyName;
+    }
+    const pathDelim = remotePath[remotePath.length - 1] === '/' || fileOnlyName[0] === '/' ? '' : '/';
+
+    const remoteFileName = remotePath && remotePath.length > 0 ? remotePath + pathDelim + fileOnlyName : fileOnlyName;
+    console.log('remoteFileName = ' + remoteFileName);
```

Comment:
Remove this console log?

Reply:
Changing them to use debug module in case something goes where we don't expect it to.